thirdparty/scripts/build_arrow.sh

```diff
@@ -58,10 +58,12 @@ if [[ ! -d $TP_DIR/../python/ray/pyarrow_files/pyarrow || \
 
     pushd $TP_DIR/build/arrow
     git fetch origin master
-    # The PR for this commit is https://github.com/apache/arrow/pull/2065. We
+    # The PR for this commit is https://github.com/apache/arrow/pull/2201. We
     # include the link here to make it easier to find the right commit because
     # Arrow often rewrites git history and invalidates certain commits.
-    git checkout ce23c06469de9cf0c3e38e35cdb8d135f341b964
+    git checkout d5d39f770047d671e4879369dd680c69afc370c3
+
+    git apply $TP_DIR/scripts/arrow-zero-fill.patch
```

Comment:
If someone want to update the arrow package, will this patch break the building process?